### PR TITLE
:bug: rewrite ProxyWindow in UIEvent

### PR DESCRIPTION
**Pull request by howel52**
##### Checklist

- [x] `npm test` passes
- [x] tests are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

- close https://github.com/umijs/qiankun/issues/570

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,11 +1,8 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es6",
```
Comment (kuitos) [inline tsconfig.json:3]: 这里为啥改成 es6 了？

File: src/sandbox/patchers/UIEvent.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      // resolve: https://github.com/umijs/qiankun/issues/570
+      // eg: https://github.com/apache/incubator-echarts/blob/master/src/component/toolbox/feature/SaveAsImage.js#L63...L75
+      mouseEventInit.view = window;
+    }
+    super(typeArg, mouseEventInit);
+  }
+}
+
+export default function patch() {
+  window.MouseEvent = FakeMouseEvent;
```
Comment (kuitos) [inline src/sandbox/patchers/UIEvent.ts:23]: 应该把 proxy 传进来作为 global，然后 global.MouseEvent = FakeMouseEvent ?

Comment (kuitos) [reply src/sandbox/patchers/UIEvent.ts:23]: 不然多实例场景就会冲突？

Comment (howel52) [reply src/sandbox/patchers/UIEvent.ts:23]: 👌

File: tsconfig.json (hunk 1 above)
Comment (howel52) [reply tsconfig.json:3]: 这里用 es6 + babel father-build 后发现 class extends + super 的写法，编译产物会有异常。

**Pull request merged**